When running in dev mode it seems to not receive auth requests (linux)

When running the browser with `yarn mock-dev`, and then trying to authorise the WHM, the request doesn't reach the browser, it seems to be that the browser is not registering correctly in linux in dev mode, the following error is thrown on the WHM shell console:
```
Error launching app
Unable to find Electron app at /home/bochaco/work/bochaco-safe_browser/app/main.js

Cannot find module '/home/bochaco/work/bochaco-safe_browser/app/main.js'
```
I also tried running the packaged browser and I was able to authorise the WHM on linux, so it's probably just the browser when launched in dev mode.

--- a/package.json
+++ b/package.json
@@ -39,7 +39,7 @@
     "flow-typed": "flow-typed install",
     "start-hot-renderer": "cross-env HOT=1 NODE_ENV=development IS_UNPACKED=true electron -r babel-register -r babel-polyfill ./app/main.development",
     "start-hot-renderer-live": "cross-env HOT=1 NODE_ENV=production IS_UNPACKED=true electron -r babel-register -r babel-polyfill ./app/main.development",
-    "postinstall": "concurrently \"yarn run build-dll\" \"yarn run build-preload\" \"yarn run build-auth-app\" \"cd app && yarn\" \"node node_modules/fbjs-scripts/node/check-dev-engines.js package.json\"",
+    "postinstall": "concurrently \"yarn run build-dll\" \"yarn run build\" \"cd app && yarn\" \"node node_modules/fbjs-scripts/node/check-dev-engines.js package.json\"",
     "dev": "yarn build-auth-app && cross-env START_HOT=1 yarn run hot-updates-server",
     "mock-dev": "yarn run dev",
     "prod-dev": "cross-env START_HOT=1 yarn run hot-updates-server-live",

